src/pages/dar_collection_review/AtAGlance.js

```diff
@@ -0,0 +1,150 @@
+import {div, h} from 'react-hyperscript-helpers';
+import {isNil, isEmpty, find, chunk} from 'lodash/fp';
+import {useState} from 'react';
+import SimpleTable from '../../components/SimpleTable';
+import {Styles} from '../../libs/theme';
+import cellData from './AtAGlanceCellData'
```

Comment:
add semi-colon to end
```suggestion
import cellData from './AtAGlanceCellData';
```